PubnativeLite/PubnativeLite/Ad Model/HyBidNativeAd.m

```diff
@@ -247,7 +257,15 @@ - (void)handleTap:(UITapGestureRecognizer *)sender {
     if (sender.state == UIGestureRecognizerStateEnded) {
         [self invokeDidClick];
         [self confirmBeaconsWithType:PNLiteNativeAdBeaconClick];
-        [[UIApplication sharedApplication] openURL:[NSURL URLWithString:self.clickUrl]];
+        
+        HyBidSkAdNetworkModel *skAdModel = [self skAdNetworkModel];
+        NSDictionary *productParameters = [skAdModel getProductParameters:skAdModel.productParameters];
+        
+        if (skAdModel && [productParameters count] != 0) {
```

Comment:
Are we checking that these params are not empty without inserting them in the dictionary?